test/libsolidity/smtCheckerTests/array_members/length_same_after_assignment_2.sol

```diff
@@ -23,3 +23,4 @@ contract C {
 // ====
 // SMTEngine: all
 // ----
+// Warning 0: (0-380): Contract invariants for :C:\n!(arr.length <= 2)\n!(arr.length <= 3)\n!(arr[2].length <= 3)\n
```

Comment:
Why `:C:` and not `C:`?

Reply:
It's the fully qualified name of the contract, but I think in isoltest the source names are empty so the contract's fully qualified name is `:C`